tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -873,20 +873,21 @@ $ sudo pip install /tmp/tensorflow_pkg/tensorflow-0.12.1-py2-none-any.whl
 ## Optimizing CPU performance
 
 To be compatible with as wide a range of machines as possible, TensorFlow
-defaults to only using SSE4.1 SIMD instructions on x86 machines. Most modern PCs
+defaults to only using SSE4 SIMD instructions on x86 machines. Most modern PCs
```

Comment:
Maybe instead: "...as possible, TensorFlow binaries only use SSE4 SIMD instructions. Most modern computers support mode advanced instructions, ..."

And, are we building with SSE4? Or only SSE3?